src/status_im/contacts/db.cljs

```diff
@@ -24,73 +24,73 @@
     (and
       (hex-string? identity)
       (or
-        (and (= 128 length) (not (str/includes? identity "0x")))
-        (and (= 130 length) (str/starts-with? identity "0x"))
-        (and (= 132 length) (str/starts-with? identity "0x04"))
+        (and (= 128 length) (not (string/includes? identity "0x")))
+        (and (= 130 length) (string/starts-with? identity "0x"))
+        (and (= 132 length) (string/starts-with? identity "0x04"))
         (is-address? identity)))))
 
-(s/def ::not-empty-string (s/and string? not-empty))
-(s/def ::public-key (s/and ::not-empty-string valid-length?))
+(spec/def ::not-empty-string (spec/and string? not-empty))
```

Comment:
I see that `::not-empty-string` is defined in multiple namespaces, this is not necessary, as spec registry is global, we should use explicitly namespaces key in this case, like `:status/not-empty-string`, define it just once (maybe in `status-im.db`) and cut the redundancy down.